jest.config.json

```diff
@@ -4,19 +4,16 @@
   "automock": false,
   "testEnvironment": "jsdom",
   "setupFiles": [
-    "<rootDir>/setupTest.js"
-  ],
-  "moduleFileExtensions": [
-    "js"
-  ],
-  "moduleDirectories": [
-    "node_modules"
+    "<rootDir>/setupTest.ts"
   ],
   "globals": {
     "__DEV__": false
   },
-  "collectCoverageFrom": [
-    "**/*.{js,jsx}"
+  "transform": {
+    "^.+\\.(t|j)sx?$": "@swc/jest"
+  },
```

Comment:
What does that do? Should we use babel and swc in the same repo? (I have no experience with swc yet, but am happy with switching to it -- but probably we should replace babel everywhere then?)

Reply:
If you want to use jest for typescipt files you need to use a proper transformer. I'm really not convinced by babel-jest, and ts-jest, as they both had their issues in the past. Also swc is about twice as fast, as it is implemented in rust :).

I decided to strip babel completely and replace it with babel.